test/integration/export_int_test.go

```diff
@@ -78,6 +79,22 @@ func (suite *ExportIntegrationTestSuite) TestExport_Config() {
 	cp.ExpectExitCode(0)
 }
 
+func (suite *ExportIntegrationTestSuite) TestExport_Env() {
+	suite.OnlyRunForTags(tagsuite.Export)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	suite.PrepareActiveStateYAML(ts)
+	asyData := fmt.Sprintf(`project: "https://platform.activestate.com/ActiveState-CLI/Export?branch=main&commitID=5397f645-da8a-4591-b106-9d7fa99545fe"`)
+	ts.PrepareActiveStateYAML(asyData)
+	cp := ts.SpawnWithOpts(
+		e2e.WithArgs("export", "env"),
+		e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
+	)
+	cp.Expect(`PATH: `)
```

Comment:
Please consider adding an additional test(s) that asserts `ACTIVESTATE_ACTIVATED` is not present, since one of the ACs is that state tool env vars should not be present. (e.g. `suite.Assert().NotContains(cp.TrimmedSnapshot(), "ACTIVESTATE_ACTIVATED")`